e2e/utils.go

```diff
@@ -37,6 +37,9 @@ const (
 
 	rookTolBoxPodLabel = "app=rook-ceph-tools"
 	rbdmountOptions    = "mountOptions"
+
+	retainPolicy = v1.PersistentVolumeReclaimRetain
+	deletePolicy = v1.PersistentVolumeReclaimDelete
```

Comment:
instead of `deletePolicy`, maybe use `defaultPolicy` or add a comment that delete should be the default for most e2e tests